tests/queries/0_stateless/02128_hex_bin_on_uuid.sql

```diff
@@ -0,0 +1,18 @@
+-- length should be 32
+select length(hex(generateUUIDv4()));
+
+with generateUUIDv4() as uuid,
+    hex(reverse(reinterpretAsString(uuid))) as str1,
+    hex(uuid) as str2
+select str1 = str2;
+
+-- hex on UUID always generate 32 characters even there're leading zeros
+select lower(hex(toUUID('00000000-80e7-46f8-0000-9d773a2fd319')));
```

Comment:
I prefer if the order of output characters will be exactly the same as in UUID:

`0000000080e746f800009d773a2fd319`

Reply:
Actually, this is how toUUID treats the input string. toUUID treats the input string as a little-endian order string.